oem/firstvoices/windows/src/xml/strings.xml

```diff
@@ -388,6 +388,11 @@
   <!-- Introduced: 7.0.230.0 -->
   <string name="koShowWelcome" comment="Startup options - show/hide welcome screen">Show welcome screen</string>
 
+  <!-- Context: Configuration Dialog - Options tab -->
+  <!-- String Type: FormatString -->
+  <!-- Introduced: 18.0.96.0 -->
+  <string name="koAutomaticUpdate" comment="Automatically download updates ready to install">Automatically download updates ready to install</string>
```

Comment:
Huh, how can it be ready to install if you haven't downloaded it?

Reply:
So I am confused, I now think the text means, "Automatically downloads updates so that they are ready to install (even when offline)", is that right?

If so, "Automatically download updates, for later installation" perhaps?